### PR TITLE
Modified table name from in_process to twitter_potential_incidents.

### PR DESCRIPTION
**Pull request by nicolascarva**
Naming: feature/descriptive-name

Reviewers: minimum 2 (Chris and/or 2 from Web or DS respectively and not person who submitted PR)

Description: 
  What is the motivation for changes?
  What specific changes were made?

Submitter Checklist: 
  - [x] Small scope (1-2 components)
  - [x] Not too many concerns in a single commit
  - [x] Remove extraneous comments (code in comments), print statements, to-dos, console.logs, extra fluff
  - [x] Atomic, descriptive commit messages
  - [x] Consistent formatting

Reviewer Checklist:
  - [x] Small scope (1-2 components)
  - [x] Comments, print statements, to-dos, console.logs, extra fluff were removed
  - [x] Give feedback no matter what conversations happened elsewhere, document here
  - [x] Comment on the line of code itself (blue plus box) in files changed tab

### CODE REVIEW

File: app/create_db_tables.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def initialize_police_table():
         confidence FLOAT
     )""",
 
-    """CREATE TABLE IF NOT EXISTS in_process (
+    """CREATE TABLE IF NOT EXISTS twitter_potential_incidents (
         id SERIAL PRIMARY KEY NOT NULL,
```
Comment (jalirkan) [inline app/create_db_tables.py:28]: This is much more understandable.

**Comment by jalirkan** [comment]
Good fix!

**Comment by njfleck24** [comment]
Beautiful

File: app/create_db_tables.py [hunk 2]
```diff
@@ -24,7 +24,7 @@ def initialize_police_table():
         confidence FLOAT
     )""",
 
-    """CREATE TABLE IF NOT EXISTS in_process (
+    """CREATE TABLE IF NOT EXISTS twitter_potential_incidents (
```
Comment (njfleck24) [inline app/create_db_tables.py:27]: So much better

**Pull request merged**